packages/backend-modules/assets/lib/getCropDimensions.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (crop) => {
+  if (!crop) {
+    return {
+      x: null,
+      y: null,
+      width: null,
+      height: null,
+    }
+  }
+  // /[^0-9]/ regex: match non-numeric values
+  // turns '10x20y100w10h' into ["10", "20", "100", "10", ""] and removes the last value
+  const [_x, _y, _width, _height] = crop.split(/[^0-9]/).slice(0, -1)
```

Comment:
```suggestion
  const [_x, _y, _width, _height] = crop.split(/[\D]/)
```

`\D` = not a digit

Destructuring does not require slicing; in your case, it will always pick first 4 array items.

Reply:
However, this regex is too broad.

Harden it e.g. string has to look like `\d+x\dy\d...` or throw harsh error.